Book-A-Desk.Domain/QueriesHandler.Reservation.fs

```diff
@@ -15,6 +16,15 @@ module rec ReservationsQueriesHandler =
         let isSameDate = isSameDate date
         return List.where (fun booking -> isSameDate booking.Date) bookings
     }
+    
+    let getUserBookingsStartFrom (bookingEvents : seq<DomainEvent>) (email : EmailAddress) (date : DateTime) : Result<Booking list, string> = result {
+        let bookingEvents =
+                bookingEvents
+                |> Seq.map (function | ReservationEvent event -> event)
+        
+        let bookings = (ReservationAggregate.getCurrentStateFrom bookingEvents).BookedDesks
+        return List.where (fun booking -> booking.EmailAddress = email && booking.Date >= date) bookings
```

Comment:
To double check: booking.EmailAddress = email might not work correctly (would compare references instead of the actual email addresses).

Probably works, just in case :)